platforms/documentation/docs/src/docs/userguide/optimizing-builds/configuration-cache/configuration_cache.adoc

```diff
@@ -0,0 +1,210 @@
+
+// Run tests for snippets included in this chapter with:
+// ./gradlew :docs:docsTest --tests="ExemplarExternalSamplesFunctionalTest.snippet-configuration-cache-*"
+// ./gradlew :docs:docsTest --tests="ExemplarExternalSamplesFunctionalTest.snippet-value-providers-*"
+
+The Configuration Cache improves build performance by caching the result of the <<build_lifecycle#build_lifecycle,configuration phase>> and reusing it for subsequent builds.
+
+image::configuration-cache-4.png[]
+
+[[config_cache:intro]]
+== Configuration Cache Basics
+
+The <<build_lifecycle.adoc#build_lifecycle, Gradle Build Lifecycle>> consists of three phases: **initialization**, **configuration**, and **execution**.
+
+1. The **initialization** phase determines the structure of the build.
+2. The **configuration** phase evaluates what work needs to be done.
+3. The **execution** phase performs that work.
+
+Gradle’s **Build Cache** has long optimized the execution phase by reusing previously built outputs and parallelizing as much work as possible.
+
+The **Configuration Cache** builds on this idea of _work avoidance_ and _parallelization_.
+When enabled, the Configuration Cache allows Gradle to skip the configuration phase entirely if nothing that affects the build configuration (such as build scripts) has changed.
+Additionally, Gradle applies performance optimizations to task execution.
+
+To do this, Gradle does the following:
+
+[cols="~,~,~,~"]
+|===
+| Phase | Cache | Work Avoidance | Parallelism and Performance
+
+| *Execution* | Build Cache | Uses task inputs and outputs | Runs independent tasks in parallel with `--parallel`
+| *Configuration* | Configuration Cache | Uses build logic and environment | Runs tasks in parallel (within the same project), optimizes memory, and more
+|===
+
+The Configuration Cache is similar to the <<build_cache#build_cache,Build Cache>>, but they store different types of data:
+
+- *Build Cache*: Stores outputs and intermediate files of the build (e.g., task outputs, artifact transform outputs).
+- *Configuration Cache*: Stores the build configuration for a particular set of tasks, capturing the output of the configuration phase.
+
+image::configuration-cache-2.png[]
+
+[IMPORTANT]
+====
+This feature is _not enabled by default_ and has the following limitations:
+
+- It does not yet support all <<configuration_cache_status.adoc#config_cache:plugins:core, Core Gradle Plugins>> and <<configuration_cache_status.adoc#config_cache:not_yet_implemented, features>>. Full support is in progress.
+- Your build and its plugins may require modifications to meet the <<configuration_cache_requirements.adoc#config_cache:requirements, Configuration Cache requirements>>.
+- <<config_cache:ide,IDE imports and syncs>> do not yet use the Configuration Cache.
+- Continuous Integration (CI) support is actively evolving.
+
+The Configuration Cache will be the preferred mode of execution in Gradle 9.0.
```

Comment:
💅 Can we change this to present tense? And maybe link to something that describes what the "preferred mode" is (I suspect, the blog post is our best source so far).
```suggestion
Since Gradle 9.0 the Configuration Cache is the [preferred mode of execution](https://blog.gradle.org/road-to-configuration-cache#preferred-mode-of-execution).
```